graph.cpp

```diff
@@ -116,10 +116,8 @@ void Graph::stop() {
 }
 
 void Graph::lockedStop() {
-    for (auto it = nodes_.begin(); it != nodes_.end(); ++it) {
-        it->second->closeConnectedPins();
-        it->second->stop();
-    }
+    for (auto it = nodes_.begin(); it != nodes_.end(); ++it) { it->second->closeConnectedPins(); }
+    for (auto it = nodes_.begin(); it != nodes_.end(); ++it) { it->second->stop(); }
```

Comment:
if we're going for c++14 here, you can also do
```cpp
for (auto& node: nodes_) { node.closeConnectedPins(); }
for (auto& node: nodes_) {node.stop(); }
```

which may be clearer

Reply:
mediagraph is at c++11, which supports your suggestion. Applying.